tests/flow/test_slowlog.py

```diff
@@ -59,3 +58,27 @@ def test_slowlog(self):
         # validate the command added to redis slowlog
         self.env.assertGreater(len(slowlog), 0)
         self.env.assertContains(b"GRAPH.QUERY slowlog_test MATCH (n), (m) WHERE n.v > 0 AND n.v < 500 SET m.v = rand() WITH n, m RETURN SUM(n.v + m.v) --compact", slowlog_commands)
+
+    def test02_slowlog_reset(self):
+        # reset none existing slowlog
+        try:
+            slowlog = redis_con.execute_command("GRAPH.SLOWLOG", "NONE_EXISTING_GRAPH", "RESET")
+        except ResponseError as e:
+            self.env.assertIn("Invalid graph operation on empty key", str(e))
+
+        # issue an unknown slowlog sub command
+        try:
+            slowlog = redis_con.execute_command("GRAPH.SLOWLOG", GRAPH_ID, "UNKNOW_SUB_CMD")
+        except ResponseError as e:
+            self.env.assertIn("Unknown subcommand", str(e))
+
+        # clear an existing slowlog
+        slowlog = redis_con.execute_command("GRAPH.SLOWLOG", GRAPH_ID)
+        self.env.assertGreater(len(slowlog), 0)
+
+        redis_con.execute_command("GRAPH.SLOWLOG", GRAPH_ID, "RESET")
+        redis_con.execute_command("GRAPH.SLOWLOG", GRAPH_ID, "RESET")
```

Comment:
why duplicate reset?

Reply:
making sure it's OK to reset an empty slowlog